src/index.js

```diff
@@ -36,18 +36,21 @@ const main = async () => {
   // track when the last frame rendered
   let lastFrameMilis = 0;
 
-  const modelRefs = [require('./models/raymanModel.obj'), require('./models/cow.obj')];
+  const modelRefs = [
+    { model: require('./models/raymanModel.obj'), name: 'rayman' }, 
+    { model: require('./models/cow.obj'), name: 'cow' }
+  ];
 
   await manager.addModels(modelRefs);
 
-  const myRayman = new GameObject(manager.modelList[0], new Physics());
-  const myCow = new GameObject(manager.modelList[1], new Physics());
+  const myRayman = new GameObject(manager.modelList['rayman'], new Physics());
+  const myCow = new GameObject(manager.modelList['cow'], new Physics());
 
```

Comment:
Small nit: you can do `const myRayman = new GameObject(manager.modelList.rayman, new Physics());` instead. Same thing for the cow model